code/modules/clothing/under/xenos/vox.dm

```diff
@@ -1,8 +1,8 @@
 /obj/item/clothing/under/vox
 	has_sensor = 0
 	species_restricted = list(SPECIES_VOX)
-	valid_accessory_slots = list("vox")
-	restricted_accessory_slots = list("vox")
+	valid_accessory_slots = "vox"
```

Comment:
Will these even work now, since it's not a bitflag?

Reply:
I was hoping you wouldn't notice thaaaaat. ;( You told me once Vox aren't really used.

I can use up a flag on them, but that onnneee item is the only thing that does that. Is it a big deal if other people can put on Vox webbing? ;v

Reply:
At Some Point ™️ there'll be a handful of OP vox accessories to make up for their lack of backpacks, so it's kinda bad.